cluster/cluster_members.go

```diff
@@ -73,18 +74,19 @@ func (c InternalClusterMember) ToAPI() (*internalTypes.ClusterMember, error) {
 			Address:     address,
 			Certificate: *certificate,
 		},
-		Role:          string(c.Role),
-		SchemaVersion: c.Schema,
-		LastHeartbeat: c.Heartbeat,
-		Status:        internalTypes.MemberUnreachable,
+		Role:                  string(c.Role),
+		SchemaInternalVersion: c.SchemaInternal,
+		SchemaExternalVersion: c.SchemaExternal,
+		LastHeartbeat:         c.Heartbeat,
+		Status:                internalTypes.MemberUnreachable,
 	}, nil
 }
 
 // UpdateClusterMemberSchemaVersion sets the schema version for the cluster member with the given address.
 // This helper is non-generated to work before generated statements are loaded, as we update the schema.
-func UpdateClusterMemberSchemaVersion(tx *sql.Tx, version int, address string) error {
-	stmt := "UPDATE internal_cluster_members SET schema=? WHERE address=?"
-	result, err := tx.Exec(stmt, version, address)
+func UpdateClusterMemberSchemaVersion(tx *sql.Tx, internalVersion uint64, externalVersion uint64, address string) error {
+	stmt := "UPDATE internal_cluster_members SET internal_schema=?,external_schema=? WHERE address=?"
```

Comment:
There looks to be a typo. The columns in the database schema are called `schema_internal` and `schema_external`.

Reply:
Ah good catch, I swapped them around yesterday but missed these ones.